src/blindBackrunDebug.sol

```diff
@@ -111,6 +111,8 @@ contract BlindBackrun is Ownable {
         uint profitToCoinbase = profit.mul(percentageToPayToCoinbase).div(100);
         IWETH(WETH_ADDRESS).withdraw(profitToCoinbase);
         block.coinbase.transfer(profitToCoinbase);
```

Comment:
Why not withdraw all of it and transfer ETH back to the caller? It's cheaper and probably what they want. We aren't trying to accumulate WETH here for larger arbitrages in the future (and in fact doing so would be risky due to the above comment re: `public`)

Reply:
I made this change.